glommio/src/executor/mod.rs

```diff
@@ -577,58 +579,24 @@ pub enum Placement {
     /// For the `Unbound` variant, the [`LocalExecutor`]s created by a
     /// [`LocalExecutorPoolBuilder`] are not bound to any CPU.
     Unbound,
-    /* MaxSpread,
-     * MaxPack,
+    /// Iterates over `CpuLocation`s in the machine topology with a high degree
+    /// of separation from previous `CpuLocation`s returned by the iterator.
+    /// The order in which items are returned by `MaxSpread` is
+    /// non-deterministic.  If the number of CPUs requested is larger than the
+    /// number of CPUs online, this `Placement` will cycle through
+    /// `CpuLocation`. Each cycle is non-deterministic meaning that
+    /// the order may differ from prior cycles.
+    MaxSpread,
```

Comment:
Should those take an `Optional` parameter with a cpu list? The idea is that you may want to spread maximally (or minimally), but only within a subset of the CPUs.

I think this is particularly important in a scenario where you are sharing the node.

Reply:
Yeah, this sounds like a good idea -- will update as well.